--- gmp-4.1.2/acinclude.m4.sopwith	2002-09-07 20:02:19.000000000 -0400
+++ gmp-4.1.2/acinclude.m4	2003-02-11 10:48:30.000000000 -0500
@@ -1003,7 +1003,7 @@
 ${tmp_underscore}underscore_test$gmp_cv_asm_label_suffix
 EOF
   case $host in
-  *-*-aix*)
+  *-*-aix*|powerpc64-*)
     cat >>conftes2.s <<EOF
 	$gmp_cv_asm_globl .${tmp_underscore}underscore_test
 .${tmp_underscore}underscore_test$gmp_cv_asm_label_suffix
--- gmp-4.1.2/configure.in.sopwith	2003-02-11 10:48:30.000000000 -0500
+++ gmp-4.1.2/configure.in	2003-02-11 10:51:58.000000000 -0500
@@ -669,6 +669,11 @@
     case $host_cpu in
       powerpc740 | powerpc750 | powerpc7400)
         path="powerpc32/750 powerpc32" ;;
+      powerpc64)
+        gcc_cflags="-g -O2"
+	gcc_cflags_asm="-Wa,-mppc64"
+	extra_functions=""
+	;;
       *)
         path="powerpc32" ;;
     esac
@@ -1943,6 +1948,8 @@
       GMP_ASM_M68K_ADDRESSING
       GMP_ASM_M68K_BRANCHES
       ;;
+    powerpc64*-*-*)
+      ;;
     powerpc*-*-*)
       GMP_ASM_POWERPC_R_REGISTERS
       GMP_INCLUDE_MPN(powerpc32/powerpc-defs.m4)
