--- gmp-4.2.4/configure.in.BAD	2009-04-09 16:57:17.000000000 -0500
+++ gmp-4.2.4/configure.in	2009-04-09 16:57:49.000000000 -0500
@@ -36,18 +36,6 @@
 m4_pattern_allow(GMP_NAIL_BITS)
 m4_pattern_allow(GMP_NUMB_BITS)
 
-# If --target is not used then $target_alias is empty, but if say
-# "./configure athlon-pc-freebsd3.5" is used, then all three of
-# $build_alias, $host_alias and $target_alias are set to
-# "athlon-pc-freebsd3.5".
-#
-if test -n "$target_alias" && test "$target_alias" != "$host_alias"; then
-  AC_MSG_ERROR([--target is not appropriate for GMP
-Use --build=CPU-VENDOR-OS if you need to specify your CPU and/or system
-explicitly.  Use --host if cross-compiling (see "Installing GMP" in the
-manual for more on this).])
-fi
-
 GMP_INIT(config.m4)
 
 AC_CANONICAL_HOST
